.gitignore

```diff
@@ -4,3 +4,4 @@ CTestTestfile.cmake
 Makefile
 bin/
 cmake_install.cmake
+.stdafx.c.pch
```

Comment:
to clarify this line. 
My editor uses this file for a kind of intellisense, but you have no need for it